grpc-all downgrade causes class not found on io.grpc.ServiceDescriptor

**kafka-connector** pom specifies to use **io-grpc:grpc-all:0.14.0** yet the other dependency, **com.google.api.grpc:pubsub:0.0.9** requires **io-grpc:grpc-core:0.15.0** since **io.grpc.ServiceDescriptor** does not exist until **0.15.0**:

**kafka-connector:master** pom:
```
<dependency>
  <groupId>io.grpc</groupId>
  <artifactId>grpc-all</artifactId>
  <version>0.14.0</version>
</dependency>
...
<dependency>
  <groupId>com.google.api.grpc</groupId>
  <artifactId>grpc-google-pubsub-v1</artifactId>
  <version>0.0.9</version>
</dependency>
```
whereas the pom for **com.google.api.grpc:pubsub:0.0.9**:
```
<dependency>
  <groupId>io.grpc</groupId>
  <artifactId>grpc-all</artifactId>
  <version>0.15.0</version>
  <scope>compile</scope>
</dependency>
```
Trying to start a **CloudPubSubGRPCPublisher** I receive:
```
Caught: java.lang.NoClassDefFoundError: io/grpc/ServiceDescriptor
java.lang.NoClassDefFoundError: io/grpc/ServiceDescriptor
	at trendGooglePubSubPitches.run(trendGooglePubSubPitches:12)
Caused by: java.lang.ClassNotFoundException: io.grpc.ServiceDescriptor
	... 1 more
```
Finally, I confirm that **grpc-core:0.14.0** does not have **ServiceDescriptor** whereas **0.15.0** does:
```
Ryan-Smalls-MacBook-Pro:kafka-connector rsmall$ jar tf ~/.m2/repository/io/grpc/grpc-core/0.14.0/grpc-core-0.14.0.jar | grep Service
io/grpc/ServerServiceDefinition$1.class
io/grpc/ServerServiceDefinition.class
io/grpc/BindableService.class
io/grpc/ServerServiceDefinition$Builder.class
```
Versus
```
Ryan-Smalls-MacBook-Pro:kafka-connector rsmall$ jar tf ~/Downloads/grpc-core-0.15.0.jar | grep Service
io/grpc/ServerServiceDefinition$1.class
io/grpc/ServiceDescriptor.class
io/grpc/ServerServiceDefinition.class
io/grpc/BindableService.class
io/grpc/ServerServiceDefinition$Builder.class
```
That **0.15.0** jar I downloaded from Maven central to inspect directly.

--- a/kafka-connector/pom.xml
+++ b/kafka-connector/pom.xml
@@ -20,7 +20,7 @@
     <dependency>
       <groupId>io.grpc</groupId>
       <artifactId>grpc-all</artifactId>
-      <version>0.14.0</version>
+      <version>0.15.0</version>
     </dependency>
     <dependency>
       <groupId>io.netty</groupId>
@@ -139,7 +139,7 @@
           -->
           <protocArtifact>com.google.protobuf:protoc:3.0.0-beta-2:exe:${os.detected.classifier}</protocArtifact>
           <pluginId>grpc-java</pluginId>
-          <pluginArtifact>io.grpc:protoc-gen-grpc-java:0.14.0:exe:${os.detected.classifier}</pluginArtifact>
+          <pluginArtifact>io.grpc:protoc-gen-grpc-java:0.15.0:exe:${os.detected.classifier}</pluginArtifact>
           <protoSourceRoot>${project.basedir}/src/main/proto</protoSourceRoot>
         </configuration>
         <executions>

